sunyar-fabric.md

```diff
@@ -26,4 +24,5 @@ _Provide the name of your sponsor. A sponsor is optional, but the sponsor must b
 
 # Pre-existing repository
 _If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**_
-- https://github.com/<your_repo>
+
+•	https://github.com/sunyar-fabric/sunyar
```

Comment:
I received a 404 when attempting to look at this repo in GitHub. Please make the repo public so that we may check things such as license and DCO sign offs. This will be required before we allow for this lab to be approved.